tests/test_shell_script_generator.py

```diff
@@ -88,7 +88,7 @@
   fi
 
   echo 'Creating custom image.'
-  gcloud compute images create my-image       --project=my-project       --source-disk-zone=us-west1-a       --source-disk=my-image-install       --storage-location=us-east1       --family=debian9
+  gcloud compute images create my-image       --project=my-project       --source-disk-zone=us-west1-a       --source-disk=my-image-install       --storage-location=us-east1       --family=debian9       --licenses='https://www.googleapis.com/compute/v1/projects/my-project/global/licenses/my-license'
```

Comment:
The family needs to be updated here.  We're up to 12 on the debian versions, and so support for 9 has slipped into the bit bucket with oldoldoldstable